### PR TITLE
Enables set different Node versions in Testkit Dockerfile

### PR DESCRIPTION
**Pull request by bigmontz**
This change enables testing the driver using different NodeJS versions.

### CODE REVIEW

File: testkit/Dockerfile [hunk 1]
```diff
@@ -41,4 +39,12 @@ USER driver
 WORKDIR /home/driver
 CMD /bin/bash
 RUN mkdir /home/driver/.npm_global
+
+# Configuring NodeJS version
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
```
Comment (fbiville) [inline testkit/Dockerfile]: you could parameterize NVM version as well, or?

File: testkit/Dockerfile [hunk 2]
```diff
@@ -41,4 +39,12 @@ USER driver
...
+RUN nvm install ${NODE_VERSION:=10}
```
Comment (fbiville) [inline testkit/Dockerfile]: probably for another PR, but Node 10 has reached EOL as you know

**Review by fbiville — APPROVED** [review]

File: testkit/Dockerfile (hunk 2 above)
Comment (bigmontz) [reply testkit/Dockerfile]: It's end-of-life, but it is the one which you could install with apt in debian/ubuntu lts.

Comment (fbiville) [reply testkit/Dockerfile]: (not important for this PR but:) for Go, I manually install a recent version, I am not installing the default, EOL one.

**Review by fbiville — APPROVED** [review]

**Pull request merged**